### ISSUE TITLE
Assess accessible approach to thumbnail images in browse lists

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#978]
We've received feedback that our current approach to accessible thumbnails is insufficient. Currently, Omeka Classic provides the filename or title of the primary media thumbnail representing a resource. Here are some possible solutions:

* Set a blank alt tag, [marking the image as a decorative portion of a text link](https://www.w3.org/WAI/tutorials/images/decorative/#example-2-decorative-image-as-part-of-a-text-link).
* Setting `aria-hidden="true"` so that screen readers skip the thumbnails altogether.
* Using the primary media's description metadata, with either of the previous two solutions as a fallback.

I would love thoughts from screen reader users or others more well-versed in accessible practices.

**Comment by sharonmleon** [comment]
I don't have anything useful to contribute here, but I figured I'd tag in @allanaaa and @mebrett for thoughts.

**Comment by zerocrates** [comment]
I'd definitely say blank alt over aria-hidden though I'm not sure we want either.

The S solution has blank alt as the default but lets users set the desired alt (or choose to show a, in classic terms, element).

**Comment by kimisgold** [comment]
Could we do something like an "Accessibility" tab for media? I'm thinking it could include a metadata field for alt text that would call attention to the 100 character limit, and maybe upload fields for video captions or subtitles. I'm thinking that using the Dublin Core description, if used, is almost always going to exceed that small limit. It could also raise awareness to web accessibility needs among our users if we call attention to it well.

**Comment by kimisgold** [comment]
I can appreciate that complexity. I was mainly thinking that having files specifically flagged as video tracks could make rendering them in the core easier.
